app/components/tf-fields/label.js

```diff
@@ -0,0 +1 @@
+export { default } from 'ember-ticketfly-fields/components/tf-fields/label';
```

Comment:
Do we want to make these "public" components? I'd be inclined to say that we don't need to include `tf-fields/label` and `tf-fields/input-text` in the `app` directory.